extension/src/helpers/metrics.ts

```diff
@@ -109,12 +125,21 @@ const getUserId = () => {
 export const emitMetric = (name: string, body?: any) => {
   const isDataSharingAllowed = settingsDataSharingSelector(store.getState());
   if (!isDataSharingAllowed) return;
+
+  const metricsData: MetricsData = JSON.parse(
+    localStorage.getItem(METRICS_DATA) || "{}",
+  );
+
   cache.push({
     /* eslint-disable camelcase */
     event_type: name,
     event_properties: body,
     user_id: getUserId(),
     device_id: window.navigator.userAgent,
+    freighter_account_funded: metricsData.freighterFunded,
```

Comment:
since we don't have a "dev" way to test this once this is deployed I can check amplitude to see if these fields are showing up right for new User Properties